frontend/apps/picasso/pages/crowdloan-rewards/claim.tsx

```diff
@@ -70,6 +71,13 @@ export const ClaimLoanPage = () => {
   const theme = useTheme();
   const hasStarted = useCrowdloanRewardsHasStarted(parachainApi);
 
+  const [ethClaimable, setEthClaimable] = useState(new BigNumber(0));
+  const [ethClaimed, setEthClaimed] = useState(new BigNumber(0));
+  const [ethTotal, setEthTotal] = useState(new BigNumber(0));
+
+  const [ksmClaimable, setKsmClaimable] = useState(new BigNumber(0));
+  const [ksmClaimed, setKsmClaimed] = useState(new BigNumber(0));
+  const [ksmTotal, setKsmTotal] = useState(new BigNumber(0));
```

Comment:
Confused by the naming here. KSM and Eth are not claimable through the UI right?

Reply:
Understandable, these names are not 100% reflective. So they are actually reflective of the query we call for claim({ethereum:...}). 

I'll present a more readable naming for these soon.